lib/core/card-view/components/card-view-textitem/card-view-textitem.component.html

```diff
@@ -1,88 +1,131 @@
-<div [attr.data-automation-id]="'card-textitem-label-' + property.key" class="adf-property-label" *ngIf="showProperty() || isEditable()">{{ property.label | translate }}</div>
+<div [attr.data-automation-id]="'card-textitem-label-' + property.key"
+     class="adf-property-label"
+     *ngIf="showProperty() || isEditable()">{{ property.label | translate }}</div>
 <div class="adf-property-value">
     <span *ngIf="!isEditable()">
-        <span *ngIf="!isClickable(); else elseBlock" [attr.data-automation-id]="'card-textitem-value-' + property.key">
-            <span *ngIf="showProperty()"
-                [ngClass]="property.multiline?'adf-textitem-multiline':'adf-textitem-scroll'">
-                {{ property.displayValue }}</span>
+        <span *ngIf="!isClickable(); else elseBlock"
```

Comment:
I know it's not part of your changes, but could you please make `elseBlock` more meaningful and rename to something like `unclickable` or similar?